buildSrc/build.gradle.kts

```diff
@@ -16,4 +16,5 @@ dependencies {
     implementation(libs.detekt.gradle)
     implementation(libs.ktlint.gradle)
     implementation(libs.benManesVersions.gradle)
+    implementation("com.squareup:javapoet:1.13.0")
```

Comment:
[Request] Please add this to `libs.versions.toml` to have all depenendencies and their versions in one place.